api/user/posts/model.py

```diff
@@ -0,0 +1,89 @@
+from datetime import datetime
+
+from flask_sqlalchemy import SQLAlchemy
+
+from api.app import db
+
+
+class Post(db.Model):
+    _id = db.Column(db.Integer, primary_key=True)
+    _published = db.Column("published", db.DateTime, nullable=False)
+    _title = db.Column("title", db.String(120), nullable=False)
+    _origin = db.Column("origin", db.Text, nullable=False)
+    _source = db.Column("source", db.Text, nullable=False)
+    _description = db.Column("short_desc", db.String(100))
+    _contentType = db.Column("contentType", db.String(50), nullable=False)
+    _content = db.Column("content", db.Text, nullable=False)
+
+    # categories will be comma separated values
+    _categories = db.Column("categories", db.Text)
+
+    # 0 -> "PUBLIC", 1-> "FRIENDS"
+    _visibility = db.Column("visibility", db.Boolean, nullable=False)
```

Comment:
I would rather have this as a short string, or better yet an enum. Nobody is ever going to remember that True visibility is friends etc. See this example:

```py
import enum
from sqlalchemy import Integer, Enum

class MyEnum(enum.Enum):
    one = 1
    two = 2
    three = 3

class MyClass(Base):
    id = Column(Integer, primary_key=True)
    value = Column(Enum(MyEnum))
```

Reply:
good suggestion, I didn't know python had enums. I ll change it